_sections/home-english.md

```diff
@@ -172,6 +174,35 @@ Click on the link to look at the complete [pizza_questionnaire](https://docs.goo
 **Caveat**  
 When you export data using this **or_other** option, in the **favorite_topping** column, you will see a value **other**. A separate column will have the answer for the questions in which the user selected **other**. This makes data analysis more cumbersome, so we do not recommend the **or_other** construct for large scale data collection efforts. See the **Relevant** section below for an alternative method more appropriate for large scale projects.
 
+### Multiple choice from file
+
+The options in a multiple choice question can also be taken from a separate file instead of the choices sheet. This is particularly useful if the options are dynamic or if the list of options is used in multiple surveys. Two types of files are supported: CSV and XML files. See usage examples below:
+
+| type                                    | name | label                          | choice_filter   |
+| --------------------------------------- | ---- | ------------------------------ | --------------- |
+| select_multiple_from_file country.csv   | dest | Which countries did you live?  |                 |
+| select_one_from_file countries.xml      | cou  | Which country do you live now? |                 |
```

Comment:
Same as above -- "**In** which country do you live now?" or "Which country do you live **in** now?"